checkov/terraform/checks/resource/kubernetes/Tiller.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class Tiller(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure that Tiller (Helm v2) is not deployed"
+        id = "CKV_K8S_34"
+        supported_resources = ["kubernetes_pod"]
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf) -> CheckResult:
+        if "metadata" in conf and isinstance(conf["metadata"], list):
+            metadata = conf.get("metadata")[0]
+
+            if metadata.get("labels") and isinstance(metadata.get("labels"), list):
```

Comment:
```suggestion
            if metadata.get("labels") and isinstance(metadata.get("labels"), list) and isinstance(metadata.get("labels")[0], dict):
```
there was an issue with the other Tiller check, probably should add the same fix here 😄